diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
index 2819045..e8f869b 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
@@ -4,24 +4,16 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+/**
+ * Patch Modifications Copyright(c) Roger Jiang
+ */
 import React from 'react';
-import clsx from 'clsx';
-import {ThemeClassNames, usePrismTheme} from '@docusaurus/theme-common';
-import {getPrismCssVariables} from '@docusaurus/theme-common/internal';
-import styles from './styles.module.css';
 export default function CodeBlockContainer({as: As, ...props}) {
-  const prismTheme = usePrismTheme();
-  const prismCssVariables = getPrismCssVariables(prismTheme);
   return (
     <As
       // Polymorphic components are hard to type, without `oneOf` generics
       {...props}
-      style={prismCssVariables}
-      className={clsx(
-        props.className,
-        styles.codeBlockContainer,
-        ThemeClassNames.common.codeBlock,
-      )}
+      className="codeBlockContainer"
     />
   );
 }
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
index 27adece..b5c0e33 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
@@ -4,11 +4,3 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-.codeBlockContainer {
-  background: var(--prism-background-color);
-  color: var(--prism-color);
-  margin-bottom: var(--ifm-leading);
-  box-shadow: var(--ifm-global-shadow-lw);
-  border-radius: var(--ifm-code-border-radius);
-}
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
index e0486b8..161f65f 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
@@ -4,6 +4,9 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+/**
+ * Patch Modifications Copyright(c) Roger Jiang
+ */
 import React from 'react';
 import clsx from 'clsx';
 import {useThemeConfig, usePrismTheme} from '@docusaurus/theme-common';
@@ -19,7 +22,6 @@ import Line from '@theme/CodeBlock/Line';
 import CopyButton from '@theme/CodeBlock/CopyButton';
 import WordWrapButton from '@theme/CodeBlock/WordWrapButton';
 import Container from '@theme/CodeBlock/Container';
-import styles from './styles.module.css';
 export default function CodeBlockString({
   children,
   className: blockClassName = '',
@@ -55,8 +57,8 @@ export default function CodeBlockString({
           !blockClassName.includes(`language-${language}`) &&
           `language-${language}`,
       )}>
-      {title && <div className={styles.codeBlockTitle}>{title}</div>}
-      <div className={styles.codeBlockContent}>
+      {title && <div className="codeBlockTitle">{title}</div>}
+      <div className="codeBlockContent">
         <Highlight
           {...defaultProps}
           theme={prismTheme}
@@ -67,12 +69,9 @@ export default function CodeBlockString({
               /* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */
               tabIndex={0}
               ref={wordWrap.codeBlockRef}
-              className={clsx(className, styles.codeBlock, 'thin-scrollbar')}>
+              className="codeBlock thin-scrollbar">
               <code
-                className={clsx(
-                  styles.codeBlockLines,
-                  showLineNumbers && styles.codeBlockLinesWithNumbering,
-                )}>
+                className={`codeBlockLines${showLineNumbers ? ` ${codeBlockLinesWithNumbering}`: ""}`}>
                 {tokens.map((line, i) => (
                   <Line
                     key={i}
@@ -87,15 +86,15 @@ export default function CodeBlockString({
             </pre>
           )}
         </Highlight>
-        <div className={styles.buttonGroup}>
+        <div className="buttonGroup">
           {(wordWrap.isEnabled || wordWrap.isCodeScrollable) && (
             <WordWrapButton
-              className={styles.codeButton}
+              className="codeButton"
               onClick={() => wordWrap.toggle()}
               isEnabled={wordWrap.isEnabled}
             />
           )}
-          <CopyButton className={styles.codeButton} code={code} />
+          <CopyButton className="codeButton" code={code} />
         </div>
       </div>
     </Container>
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/styles.module.css b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/styles.module.css
index 3760c53..e69de29 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/styles.module.css
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/styles.module.css
@@ -1,87 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-.codeBlockContent {
-  position: relative;
-  /* rtl:ignore */
-  direction: ltr;
-  border-radius: inherit;
-}
-
-.codeBlockTitle {
-  border-bottom: 1px solid var(--ifm-color-emphasis-300);
-  font-size: var(--ifm-code-font-size);
-  font-weight: 500;
-  padding: 0.75rem var(--ifm-pre-padding);
-  border-top-left-radius: inherit;
-  border-top-right-radius: inherit;
-}
-
-.codeBlock {
-  --ifm-pre-background: var(--prism-background-color);
-  margin: 0;
-  padding: 0;
-}
-
-.codeBlockTitle + .codeBlockContent .codeBlock {
-  border-top-left-radius: 0;
-  border-top-right-radius: 0;
-}
-
-.codeBlockStandalone {
-  padding: 0;
-}
-
-.codeBlockLines {
-  font: inherit;
-  /* rtl:ignore */
-  float: left;
-  min-width: 100%;
-  padding: var(--ifm-pre-padding);
-}
-
-.codeBlockLinesWithNumbering {
-  display: table;
-  padding: var(--ifm-pre-padding) 0;
-}
-
-@media print {
-  .codeBlockLines {
-    white-space: pre-wrap;
-  }
-}
-
-.buttonGroup {
-  display: flex;
-  column-gap: 0.2rem;
-  position: absolute;
-  /* rtl:ignore */
-  right: calc(var(--ifm-pre-padding) / 2);
-  top: calc(var(--ifm-pre-padding) / 2);
-}
-
-.buttonGroup button {
-  display: flex;
-  align-items: center;
-  background: var(--prism-background-color);
-  color: var(--prism-color);
-  border: 1px solid var(--ifm-color-emphasis-300);
-  border-radius: var(--ifm-global-radius);
-  padding: 0.4rem;
-  line-height: 0;
-  transition: opacity var(--ifm-transition-fast) ease-in-out;
-  opacity: 0;
-}
-
-.buttonGroup button:focus-visible,
-.buttonGroup button:hover {
-  opacity: 1 !important;
-}
-
-:global(.theme-code-block:hover) .buttonGroup button {
-  opacity: 0.4;
-}
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
index 026df2c..38ad0c3 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
@@ -11,7 +11,7 @@ export default function DocItemMetadata() {
   const {metadata, frontMatter, assets} = useDoc();
   return (
     <PageMetadata
-      title={metadata.title}
+      title={frontMatter.meta_title}
       description={metadata.description}
       keywords={frontMatter.keywords}
       image={assets.image ?? frontMatter.image}
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
index 270910e..a8cdff8 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
@@ -11,7 +11,9 @@ export default function FooterLayout({style, links, logo, copyright}) {
     <footer
       className={clsx('footer', {
         'footer--dark': style === 'dark',
-      })}>
+      })}
+      style={{height:"210px", width:"100%"}}
+      >
       <div className="container container-fluid">
         {links}
         {(logo || copyright) && (
diff --git a/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx b/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
index bb22f31..e9f1164 100644
--- a/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
+++ b/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
@@ -17,6 +17,7 @@ export default function FooterLayout({
 }: Props): JSX.Element {
   return (
     <footer
+      style={{height:"210px", width:"100%"}}
       className={clsx('footer', {
         'footer--dark': style === 'dark',
       })}>
