diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
index 2819045..e8f869b 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/index.js
@@ -4,24 +4,16 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+/**
+ * Patch Modifications Copyright(c) Roger Jiang
+ */
 import React from 'react';
-import clsx from 'clsx';
-import {ThemeClassNames, usePrismTheme} from '@docusaurus/theme-common';
-import {getPrismCssVariables} from '@docusaurus/theme-common/internal';
-import styles from './styles.module.css';
 export default function CodeBlockContainer({as: As, ...props}) {
-  const prismTheme = usePrismTheme();
-  const prismCssVariables = getPrismCssVariables(prismTheme);
   return (
     <As
       // Polymorphic components are hard to type, without `oneOf` generics
       {...props}
-      style={prismCssVariables}
-      className={clsx(
-        props.className,
-        styles.codeBlockContainer,
-        ThemeClassNames.common.codeBlock,
-      )}
+      className="codeBlockContainer"
     />
   );
 }
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
index 27adece..b5c0e33 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Container/styles.module.css
@@ -4,11 +4,3 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-.codeBlockContainer {
-  background: var(--prism-background-color);
-  color: var(--prism-color);
-  margin-bottom: var(--ifm-leading);
-  box-shadow: var(--ifm-global-shadow-lw);
-  border-radius: var(--ifm-code-border-radius);
-}
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
index e0486b8..c5bbd3b 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/CodeBlock/Content/String.js
@@ -4,6 +4,9 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+/**
+ * Patch Modifications Copyright(c) Roger Jiang
+ */
 import React from 'react';
 import clsx from 'clsx';
 import {useThemeConfig, usePrismTheme} from '@docusaurus/theme-common';
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
index 026df2c..3847ba2 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/DocItem/Metadata/index.js
@@ -11,7 +11,7 @@ export default function DocItemMetadata() {
   const {metadata, frontMatter, assets} = useDoc();
   return (
     <PageMetadata
-      title={metadata.title}
+      title={metadata.description}
       description={metadata.description}
       keywords={frontMatter.keywords}
       image={assets.image ?? frontMatter.image}
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
index 270910e..a8cdff8 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/Footer/Layout/index.js
@@ -11,7 +11,9 @@ export default function FooterLayout({style, links, logo, copyright}) {
     <footer
       className={clsx('footer', {
         'footer--dark': style === 'dark',
-      })}>
+      })}
+      style={{height:"210px", width:"100%"}}
+      >
       <div className="container container-fluid">
         {links}
         {(logo || copyright) && (
diff --git a/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx b/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
index bb22f31..e9f1164 100644
--- a/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
+++ b/node_modules/@docusaurus/theme-classic/src/theme/Footer/Layout/index.tsx
@@ -17,6 +17,7 @@ export default function FooterLayout({
 }: Props): JSX.Element {
   return (
     <footer
+      style={{height:"210px", width:"100%"}}
       className={clsx('footer', {
         'footer--dark': style === 'dark',
       })}>
